.github/dependabot.yml

```diff
@@ -0,0 +1,12 @@
+version: 2
+updates:
+  - package-ecosystem: "npm"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+      day: "sunday"
+      time: "02:00"
```

Comment:
I think that `time` is optional.
And we don't mind any time during sunday.
The purpose is to have the PRs created before coming the Monday morning

Reply:
I actually wanted to set Monday at 2 AM and messed up. ^^ But Sunday at any time works too, true.